### PR TITLE
Author CBT FINAL QA LIST

### PR DESCRIPTION
**Pull request by NohYujeong**

### CODE REVIEW

File: app/components/authorShowHeader/authorShowHeader.scss [hunk 1]
```diff
@@ -134,16 +135,14 @@
 .contactIconWrapper {
   display: inline-flex;
   width: 24px;
-  height: 24px;
+  vertical-align: middle;
```
Comment (TylorShin) [inline app/components/authorShowHeader/authorShowHeader.scss]: This is flex-layout element.
If you want to make centered vertical-align one, try to use `align-items: center`

Comment (TylorShin) [reply app/components/authorShowHeader/authorShowHeader.scss]: https://css-tricks.com/snippets/css/a-guide-to-flexbox/
this is the ultimate guide of Flexbox layout API.

File: app/components/authorShowHeader/index.tsx [hunk 2]
```diff
@@ -56,21 +56,21 @@ class AuthorShowHeader extends React.PureComponent<AuthorShowHeaderProps, Author
...
-                      <span className={styles.metricValue}>{author.hIndex}</span>
+                      <span className={styles.metricValue}>{author.hIndex.toLocaleString()}</span>
                       <span className={styles.metricLabel}>H-index</span>
                     </div>
                   )}
 
                   {(author.citationCount || author.hIndex === 0) && (
                     <div className={styles.metricWrapper}>
-                      <span className={styles.metricValue}>{author.citationCount}</span>
+                      <span className={styles.metricValue}>{author.citationCount.toLocaleString()}</span>
```
Comment (TylorShin) [inline app/components/authorShowHeader/index.tsx]: For the above three changes, I wonder why did you use `toLocaleString` method. `toLocaleString` method aims `Date` type. if you want to change them to String type object, you can use `toString()` method.

File: app/components/common/autoSizeTextarea/autoSizeTextarea.scss [hunk 3]
```diff
@@ -11,4 +11,8 @@
   margin: 0;
   padding: 13px 15.5px;
   resize: none;
+
+  &::placeholder {
+    color: $gray4;
+  }
```
Comment (TylorShin) [inline app/components/common/autoSizeTextarea/autoSizeTextarea.scss]: 👍

File: app/components/common/coAuthor/index.tsx [hunk 4]
```diff
@@ -29,7 +29,7 @@ const CoAuthor = (props: CoAuthorProps) => {
         <div className={styles.coAuthorName}>
           {author.name}{" "}
           {author.isLayered ? (
-            <div className={styles.contactIconWrapper}>
+            <div title="Verification Author" className={styles.contactIconWrapper}>
```
Comment (TylorShin) [inline app/components/common/coAuthor/index.tsx]: Is this for tooltip? let me know what `title` attribute does.

File: app/components/common/paperItem/authors.tsx [hunk 5]
```diff
@@ -6,6 +6,7 @@ import { withStyles } from "../../../helpers/withStylesHelper";
 import { trackEvent } from "../../../helpers/handleGA";
 import GlobalDialogManager from "../../../helpers/globalDialogManager";
 import { Paper } from "../../../model/paper";
+import * as classNames from "classnames";
```
Comment (TylorShin) [inline app/components/common/paperItem/authors.tsx]: Make import sentence that importing the external library be above the local module importing sentences.

File: app/components/dialog/components/modifyProfile/index.tsx [hunk 6]
```diff
@@ -113,22 +113,21 @@ class ModifyProfileDialog extends React.PureComponent<
...
+                Short Bio<small> (Optional)</small>
               </label>
               <Field
                 name="bio"
                 component={ReduxAutoSizeTextarea}
                 type="text"
                 textareaClassName={styles.textAreaWrapper}
                 textareaStyle={{ padding: "8px" }}
-                placeholder="Please tell us about yourself."
+                placeHolder="Please tell us about yourself."
```
Comment (TylorShin) [inline app/components/dialog/components/modifyProfile/index.tsx]: `placeholder` is the right attribute.

    interface InputHTMLAttributes<T> extends HTMLAttributes<T> {
        accept?: string;
        alt?: string;
        autoComplete?: string;
        autoFocus?: boolean;
        capture?: boolean | string; // https://www.w3.org/TR/html-media-capture/#the-capture-attribute
        checked?: boolean;
        crossOrigin?: string;
        disabled?: boolean;
        form?: string;
        formAction?: string;
        formEncType?: string;
        formMethod?: string;
        formNoValidate?: boolean;
        formTarget?: string;
        height?: number | string;
        list?: string;
        max?: number | string;
        maxLength?: number;
        min?: number | string;
        minLength?: number;
        multiple?: boolean;
        name?: string;
        pattern?: string;
        **placeholder?: string;**
        readOnly?: boolean;
        required?: boolean;
        size?: number;
        src?: string;
        step?: number | string;
        type?: string;
        value?: string | string[] | number;
        width?: number | string;

        onChange?: ChangeEventHandler<T>;
    }

File: app/components/authorShowHeader/index.tsx (hunk 2 above)
Comment (TylorShin) [reply app/components/authorShowHeader/index.tsx]: I've got it why you tried to use `toLocaleString` method.
You can use `formatNumber` helper method in `app/helpers/formatNumber.tsx`

File: app/components/dialog/components/modifyProfile/index.tsx (hunk 6 above)
Comment (NohYujeong) [reply app/components/dialog/components/modifyProfile/index.tsx]: The name of the placeholder was defined as placeHolder in the props interface of ReactAutoSizeTextarea.

~~~
interface ReduxAutoSizeTextareaProps extends WrappedFieldProps {
  wrapperClassName?: string;
  wrapperStyle?: React.CSSProperties;
  textareaStyle?: React.CSSProperties;
  textareaClassName?: string;
  placeHolder?: string;
  rows?: number;
  disabled: boolean;
}
~~~

 But I agree that we should unify the spelling. I'll make that change.

Comment (TylorShin) [reply app/components/dialog/components/modifyProfile/index.tsx]: 👍 👍 👍

File: app/components/common/coAuthor/index.tsx [hunk 7]
```diff
@@ -5,6 +5,7 @@ import { trackEvent } from "../../../helpers/handleGA";
 import HIndexBox from "../hIndexBox";
 import { withStyles } from "../../../helpers/withStylesHelper";
 import Icon from "../../../icons";
+import { Tooltip } from "@material-ui/core";
```
Comment (TylorShin) [inline app/components/common/coAuthor/index.tsx]: Please change all this import syntax to `import Tooltip  from "@material-ui/core/Tooltip";`
the above one can make Webpack do 'tree shaking' import to reduce a bundle size.

If you want to know more information, follow the below link!
https://webpack.js.org/guides/tree-shaking/

**Comment by TylorShin** [comment]
🍶

**Pull request merged**